src/models/Scholarships.ts

```diff
@@ -100,6 +98,13 @@ class Scholarships extends FirestoreCollection<ScholarshipData> {
 
     return FirestoreCollection.list(query, postProcessFilter);
   }
+
+  save(data: ScholarshipData): Promise<Model<ScholarshipData>> {
+    const Data = { ...data, lastModified: new Date() };
```

Comment:
[23.2](https://github.com/airbnb/javascript#naming--camelCase) Use camelCase when naming objects, functions, and instances.

```suggestion
    const newData = { ...data, lastModified: new Date() };
```